src/tools/clippy/tests/ui/new_without_default.stderr

```diff
@@ -31,6 +32,7 @@ LL +     fn default() -> Self {
 LL +         Self::new()
 LL +     }
 LL + }
+LL | impl Bar {
```

Comment:
Similarly, why is this still being printed?

Reply:
Because those are in clippy, need to rerun :)